sdk/websites/Azure.ResourceManager.AppService/src/Customization/RestOperations/WebAppsRestOperations.cs

```diff
@@ -156,5 +156,165 @@ public Response<AppServiceConfigurationDictionary> ListFunctionKeys(string subsc
                     throw new RequestFailedException(message.Response);
             }
         }
+        /// <summary> Description for Retrieves all Service Bus Hybrid Connections used by this Web App. </summary>
+        /// <param name="subscriptionId"> Your Azure subscription ID. This is a GUID-formatted string (e.g. 00000000-0000-0000-0000-000000000000). </param>
+        /// <param name="resourceGroupName"> Name of the resource group to which the resource belongs. </param>
+        /// <param name="name"> The name of the web app. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="subscriptionId"/>, <paramref name="resourceGroupName"/> or <paramref name="name"/> is null. </exception>
+        /// <exception cref="ArgumentException"> <paramref name="subscriptionId"/>, <paramref name="resourceGroupName"/> or <paramref name="name"/> is an empty string, and was expected to be non-empty. </exception>
+        public async Task<Response<HybridConnectionData>> ListHybridConnectionsAsync(string subscriptionId, string resourceGroupName, string name, CancellationToken cancellationToken = default)
```

Comment:
since this is completely wrong, I think we do not have to move this to customization code.
We could just remove this, and let the old method to call the working version and returns the first item
cc @ArthurMa1978  what do you think?

Reply:
since this method doesn't work and add obsolete attribute, just throw an exception for methods that invoke this method.